src/api/CortexApi.ts

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 import { Entity } from '@backstage/catalog-model';
-import { CustomMapping } from "@cortexapps/backstage-plugin-extensions";
+import { CustomMapping, TeamOverrides } from "@cortexapps/backstage-plugin-extensions";
 
 export interface CortexApi {
   syncEntities(
     entities: Entity[],
     customMappings?: CustomMapping[],
+    groupOverrides?: TeamOverrides,
```

Comment:
Are we purposely conflating "team overrides" and "group overrides?" If so, why?

Reply:
yeahh it's pretty confusing, it was intentional, but i'll change it